test/functional/dbcrash.py

```diff
@@ -64,7 +64,7 @@ def set_test_params(self):
 
     def setup_network(self):
         # Need a bit of extra time for the nodes to start up for this test
-        self.add_nodes(self.num_nodes, timewait=90)
+        self.add_nodes(self.num_nodes, extra_args=self.extra_args, timewait=90)
```

Comment:
We might as well get rid of passing member variables to methods.

Reply:
Agree, but I haven't looked into how big a change that would be (I'd need to look at all calls to `add_nodes()` to make sure they're not calling this method with no `extra_args` method and expecting the node to be started with any `extra_args` equal to `[]`)